docker/linea-mainnet/docker-compose.yml

```diff
@@ -69,3 +69,20 @@ services:
       - ./besu-genesis.json:/var/lib/besu/genesis.json:ro
       - ../config/linea-besu-sequencer/log4j.xml:/var/lib/besu/log4j.xml
       - ./static-nodes.json:/var/lib/besu/static-nodes.json
+
+  nethermind-node:
+    image: nethermindeth/nethermind:master
```

Comment:
```suggestion
    image: nethermindeth/nethermind:1.29.1
```

Reply:
We prefer to stick to fixed images to be intentional when updating nodes.

Reply:
This is while we don't have it as a stable release. That's when I will change it from draft to ready and update it with the expected tag. Thanks for pointing out anyway!